bitnami/airflow/templates/web/deployment.yaml

```diff
@@ -0,0 +1,237 @@
+            {{- include "airflow.configure.database" . | nindent 12 }}
+            {{- include "airflow.configure.redis" . | nindent 12 }}
+            {{- include "airflow.configure.airflow.kubernetesExecutor" . | nindent 12 }}
+            - name: AIRFLOW_USERNAME
+              value: {{ .Values.auth.username }}
+            - name: AIRFLOW_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "airflow.secretName" . }}
+                  key: airflow-password
+            {{- if .Values.web.baseUrl }}
+            - name: AIRFLOW_BASE_URL
+              value: {{ .Values.web.baseUrl }}
+            {{- end }}
+            - name: AIRFLOW_LDAP_ENABLE
+              value: {{ ternary "yes" "no" .Values.ldap.enabled | quote }}
+            {{- if .Values.ldap.enabled }}
+            - name: AIRFLOW_LDAP_URI
+              value: {{ .Values.ldap.uri }}
+            - name: AIRFLOW_LDAP_SEARCH
+              value: {{ .Values.ldap.base }}
+            - name: AIRFLOW_LDAP_BIND_USER
+              value: {{ .Values.ldap.binddn }}
+            - name: AIRFLOW_LDAP_BIND_PASSWORD
+              value: {{ .Values.ldap.bindpw }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "airflow.ldapSecretName" . }}
+                  key: bind-password
+            - name: AIRFLOW_LDAP_UID_FIELD
+              value: {{ .Values.ldap.uidField }}
+            - name: AIRFLOW_LDAP_USE_TLS
+              value: {{ ternary "yes" "no" .Values.ldap.tls.enabled | quote }}
+            {{- if .Values.ldap.tls.enabled }}
+            - name: AIRFLOW_LDAP_ALLOW_SELF_SIGNED
+              value: {{ .Values.ldap.tls.allowSelfSigned }}
+            - name: AIRFLOW_LDAP_TLS_CA_CERTIFICATE
+              value: {{ include "airflow.ldapCAFilename" . | quote }}
+            {{- end }}
+            {{- end }}
+            {{- if .Values.extraEnvVars }}
+              {{- toYaml .Values.extraEnvVars | nindent 12 }}
+            {{- end }}
+            {{- if .Values.web.extraEnvVars }}
+              {{- include "common.tplvalues.render" (dict "value" .Values.web.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.extraEnvVarsSecret }}
+            {{- end }}
+            {{- if .Values.web.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.web.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.web.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.web.extraEnvVarsSecret }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 8080
```

Comment:
We should add something like `airflow.containerPorts.http` so users can specify custom ports. This is another thing we have been doing with standarizations.

Reply:
done!